README.md

```diff
@@ -510,6 +510,20 @@ A list of tool extensions for .NET Core Command Line (dotnet CLI), aka '.NET Cor
         <code>dotnet tool install -g <a href="https://www.nuget.org/packages/dotnet-encrypto">dotnet-encrypto</a></code>
       </td>
     </tr>
+    <tr>
+      <td><code>dotnet-exec</code></td>
```

Comment:
The actual command to use your tool is just `execute`, not `dotnet-exec`.
https://github.com/ivanparvaresh/dotnet-exec/blob/master/src/dotnet-exec.csproj#L12

Reply:
Thanks, Updated

Reply:
@ivanpaulovich you need to push new commits to https://github.com/ivanparvaresh/dotnet-tools/tree/dotnet-exec. The pull request hasn't updated with your changes.

Reply:
@natemcmaster oh, you're right, updated